main.js

```diff
@@ -1,15 +1,96 @@
 function mobileMenu() {
-    const nonDisplay = document.getElementById('mobile-menu')
-    const blur = document.getElementById('blur')
-    const projects = document.getElementById('projects')
-    const about = document.getElementById('about')
-    const contact = document.getElementById('contact')
-
-   
-    blur.classList.toggle("blur")
-    nonDisplay.classList.toggle('non-display')
-    projects.classList.toggle("blur")
-    about.classList.toggle("blur")
-    contact.classList.toggle("blur")
-   }
-   
+  const nonDisplay = document.getElementById('mobile-menu');
+  const blur = document.getElementById('blur');
+  const projects = document.getElementById('projects');
+  const about = document.getElementById('about');
+  const contact = document.getElementById('contact');
+
+  blur.classList.toggle('blur');
+  nonDisplay.classList.toggle('non-display');
+  projects.classList.toggle('blur');
+  about.classList.toggle('blur');
+  contact.classList.toggle('blur');
+}
+
+const menu = document.querySelector('.menu');
+menu.addEventListener('click', mobileMenu);
+
+//    Start Modal Box
+const projectDetails = [
+  {
+    name: 'Tonic',
+    featured: ['CANOPY', 'Back End Dev', '2022'],
+    description:
+      'Loreme ipsum dolor sit, amet consectetur adipisicing elit. Dolores mollitia, sapiente deleniti voluptas exercitationem dolore, nihil facilis doloremque dicta dignissimos, fugit maxime culpa obcaecati nisi corrupti id sunt vel sequi. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Exercitationem iure pariatur dolores, voluptate natus modi vel architecto enim accusantium accusamus sed sapiente at laborum quas quo quam? Corporis, ab dolorem! NAJI',
+    image: './assest/img/Snapshoot Portfolio.png',
+    technologies: ['html', 'css', 'havascript', 'github', 'ruby', 'Bootstrap'],
+    live: 'https://naji-97.github.io/Portfolio-/',
+    source: 'https://github.com/naji-97',
+  },
```

Comment:
- [ ] In the top right corner, the featured should be a list. There should be a list icon before the last two items. Kindly check the Figma design and fix it.

- [ ] When the popup appears, the background of the page should be blurred. Kindly fix the color to match the Figma design.

- [ ] Please increases the gap between the popup and the main page border for the top, left, bottom, and right side as well. You can adjust the margin or padding to fix the issue.

- [ ] You should increase the gap between the project details and technologies and kindly match the border radius for the see live and see source buttons as the Figma design.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![your1](https://user-images.githubusercontent.com/70483835/210591115-702823f4-df02-40f4-937a-038b69a01f46.png)  |  ![figma1](https://user-images.githubusercontent.com/70483835/210591159-a15bc93e-a93f-4b88-97f9-795b99c4d99c.png)